features/autorisatie/personen/gezag-persoon-beperkt/adressering/autorisatie.feature

```diff
@@ -66,3 +66,64 @@ Regel: Vragen met fields om een adres in adressering wanneer de gebruiker niet g
     | adressering                       | 81110 81120 81130 81140 81150 81160 81170 81210 81330 81340 81350 PANM03 PANM04 PANM05 PANM06 PAVP01 PAVP02 PAVP03 PAVP04 | land (81310)         |
     | adresseringBinnenland             | 81110 81120 81130 81140 81150 81160 81170 81210 PANM03 PANM04 PANM05 PANM06 PAVP01 PAVP02 PAVP04                          | adresregel1 (PAVP03) |
     | adresseringBinnenland             | 81110 81120 81130 81140 81150 81160 81170 81210 PANM03 PANM04 PANM05 PANM06 PAVP01 PAVP02 PAVP03                          | adresregel2 (PAVP04) |
+
+Regel: Een afnemer geautoriseerd voor de virtuele rubriek 'adressering' (AD.01) mag alle adres velden van adressering opvragen
```

Comment:
Je hebt het in de features over AD.01 en AD.02. Dat moet zijn PA.AD.01 en PA.AD.02. In de autorisatietabel staat dan PAAD01 of PAAD02.

Reply:
gecorrigeerd